media-gfx/openvdb/openvdb-11.0.0.ebuild

```diff
@@ -0,0 +1,247 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+
+LLVM_MAX_SLOT=14
```

Comment:
Openvdb 11 added support for llvm 15:
https://www.openvdb.org/documentation/doxygen/changes.html#v11_0_0_changes

I would bump this to 15 and also make the llvm RDEPEND use the LLVM_MAX_SLOT variable instead of the hardcoded value it currently uses.

Reply:
I didn't really look deep into ax, because the flag is masked. But will do.